squares_x: 11
squares_y: 7
square_length: 0.035
marker_length: 0.025
dictionary: "DICT_4X4_50"
